esp/esp/program/models/class_.py

```diff
@@ -1111,7 +1115,27 @@ def firstBlockEvent(self):
     def isFull(self, ignore_changes=False, webapp=False):
         if len(self.get_meeting_times()) == 0:
             return True
-        if webapp and Tag.getBooleanTag('count_checked_in_only', program = self.parent_program, default = False):
+
+        now = datetime.datetime.now()
+        switch_time = None
+        if Tag.getProgramTag('switch_time_program_attendance', program=self.parent_program):
+            try:
+                switch_hour, switch_min = [int(x) for x in Tag.getProgramTag('switch_time_program_attendance', program=self.parent_program).split(":")]
+                switch_time = datetime.datetime(now.year, now.month, now.day, switch_hour, switch_min)
```

Comment:
Instead of doing this precarious splitting, probably cleaner to use strptime(): https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior